.github/workflows/CI.yml

```
@@ -7,7 +7,7 @@ jobs:
       matrix:
         os: [ macos-latest, ubuntu-latest, windows-latest ]
         node-version: [ 18, 20, 22 ]
-        eslint-version: [ 8, '' ]
+        eslint-version: [ 9 ]
```

If we change it as it is now, eslint@9 will be [reinstalled](https://github.com/bpmn-io/eslint-plugin-bpmn-io/pull/20/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R33) each time CI executes. The version list is necessary for backwards compatibility testing. Alternatively, we could remove the "Install eslint" step and the commented line as we don't test backwards compatibility anymore.

```suggestion
        eslint-version: [ '' ]
```

---

:+1: